src/System.Reflection.Metadata/src/System/Reflection/Metadata/Handles.cs

```diff
@@ -652,6 +652,18 @@ public override int GetHashCode()
         {
             return left._rowId != right._rowId;
         }
+
+        /// <summary>
+        /// Returns a handle to <see cref="MethodDebugInformation"/> corresponding to this handle.
+        /// </summary>
+        /// <remarks>
+        /// The resulting handle is only valid within the context of a <see cref="MetadataReader"/> open on the Portable PDB blob,
+        /// which in case of standalone PDB file is a different reader than the one containing this method definition.
+        /// </remarks>
+        public MethodDebugInformationHandle ToDebugInformationHandle()
```

Comment:
Sorry I missed the fact that this method is on MethodDefinitionHandle as opposed to Handle like I thought. However my comment still stands as I think it would be better to have this as an extension method or a FromMethodDefintion method on MethodDebugInformationHandle.